docs/content/design-keuzes.md

```diff
@@ -0,0 +1,37 @@
+# Designkeuzes
+12345678901234567890123456789012345678901234567890123456789012345678901234567890
```

Comment:
wat is dit?

Reply:
Oops, dat is mijn 80 char regel. Mag weg